cloud/aws/services/cloudformation/bootstrap.go

```diff
@@ -20,6 +20,7 @@ import (
 	"github.com/awslabs/goformation/cloudformation"
```

Comment:
Should we move the cloudformation module outside of `services`? As of right now this is used only in the helper module and it might create some confusion having in the cloud/aws/services package.